src/clusterfuzz/_internal/tests/appengine/handlers/cron/project_setup_test.py

```diff
@@ -1700,6 +1700,16 @@ def _mock_read_data(path):
             'sanitizers': ['address'],
             'platform': 'ANDROID_X86',
             'queue_id': 'pixel8'
+        }, {
+            'build_path':
+                'gs://bucket-android/a-b-android/%ENGINE%/%SANITIZER%/'
+                '%TARGET%/([0-9]+).zip',
+            'name': 'android_mte',
+            'fuzzing_engines': ['libfuzzer'],
+            'architectures': ['arm'],
+            'sanitizers': ['none'],
+            'platform': 'ANDROID_MTE',
```

Comment:
The new platform `ANDROID_MTE` is required, otherwise other android devices running on platform `ANDROID` will query the unsanitized builds.